examples/pytorch_sparse_quantized_transfer_learning/README.md

```diff
@@ -0,0 +1,94 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# PyTorch Sparse-Quantized Transfer Learning with SparseML
+
```

Comment:
grounding this with a product flow image would work well I think as discussed in slack, adding the file to this comment for the diagram as discussed

[flow-sparsification-model_sparsezoo-recipe_sparsezoo-sparsifcation_sparseml_training-deployment_deepsparse.svg.zip](https://github.com/neuralmagic/sparseml/files/6043774/flow-sparsification-model_sparsezoo-recipe_sparsezoo-sparsifcation_sparseml_training-deployment_deepsparse.svg.zip)